web/src/modules/deployments/index.ts

```diff
@@ -258,6 +273,16 @@ export const deploymentsSlice = createSlice({
         ) {
           state.canceling[action.payload.deploymentId] = false;
         }
+        if (
+          action.payload.type === Command.Type.SKIP_STAGE &&
+          (action.payload.status === CommandStatus.COMMAND_FAILED ||
+            action.payload.status === CommandStatus.COMMAND_TIMEOUT)
+        ) {
+          state.skippable[action.payload.stageId] = false;
```

Comment:
I think with the name `skippable` we should reverse the condition. Which means in case of failure or timeout, the stage should be skippable (this set should be = true) and at L285, we will set the value to false (make stage not skippable in case of click to skip button recently and in processing). WDYT?

Reply:
Oops, I mistook this one. let me fix it 🙌 
Thank you.